projects/golang/go/1.20/README.md

```diff
@@ -1,10 +1,10 @@
 # EKS Golang 1.20
 
-Current Release: `4`
+Current Release: `9`
 
-Tracking Tag: `go1.20.5`
+Tracking Tag: `go1.20.8`
```

Comment:
were these just not updated last time?

Reply:
They haven't been being update it appears.